apps/website/.vuepress/config.js

```diff
@@ -141,4 +135,5 @@ module.exports = {
       },
     },
   },
+  shouldPrefetch: () => false,
```

Comment:
One more thing: this turns off the vuepress behavior that adds prefetch links for all_the_things generated in the website. It doesn't make sense to have this _and_ a service worker that caches the same files. It is a function though so it could be customized to have a more granular approach that combines some prefetch with some service worker caching.